src/constants/aggregationFields.js

```diff
@@ -141,16 +175,23 @@ export const aggregationFields = {
         ]
     },
     ocr: {
+        category: 'ocr',
+        categoryLabel: 'OCR',
+        categoryIcon: <Translate />,
         filterLabel: 'OCRed',
         chipLabel: 'OCRed',
         type: 'term-or',
     },
     attachments: {
+        category: 'email',
+        categoryLabel: 'Email',
         filterLabel: 'Has Attachments',
         chipLabel: 'Has Attachments',
         type: 'term-or',
     },
     pgp: {
+        category: 'email',
+        categoryLabel: 'Email',
         filterLabel: 'PGP Encrypted',
         chipLabel: 'PGP Encrypted',
         type: 'term-or',
```

Comment:
reminder -- move the duplicated `categoryLabel` and `categoryIcon` to a different constant list in `src/constants`